$DragonFly: src/lib/libarchive/patches/Attic/archive_platform.h.patch,v 1.1 2004/11/10 18:02:24 joerg Exp $

--- archive_platform.h.orig	2004-11-10 00:40:28.000000000 +0100
+++ archive_platform.h	2004-11-10 00:44:11.000000000 +0100
@@ -83,6 +83,8 @@
 #define	HAVE_SYS_TYPES_H 1
 #define	HAVE_SYS_WAIT_H 1
 #define	HAVE_UNISTD_H 1
+#define	HAVE_VA_COPY 1
+#define	HAVE_VASPRINTF 1
 #define	HAVE_WCHAR_H 1
 #define	HAVE_ZLIB_H 1
 #define	STDC_HEADERS 1
